packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/web-services/invoke-contract-endpoint.ts

```diff
@@ -26,7 +26,7 @@ export class InvokeContractEndpoint implements IWebServiceEndpoint {
 
   private readonly log: Logger;
 
-  public get className() {
+  public get className(): string {
```

Comment:
@AzaharaC Thanks for fixing the occasional linter warning! It matters in the long run.